lib/improver/cli/orographic_enhancement.py

```diff
@@ -168,21 +163,16 @@ def process(temperature, humidity, pressure, wind_speed, wind_dir, orography):
             resolution (1 km) UKPP domain grid.
 
     Returns:
-        (tuple): tuple containing:
-                **orogenh_high_res** (iris.cube.Cube):
-                    Precipitation enhancement due to orography in mm/h on the
-                    UK standard grid, padded with masked up np.nans where
-                    outside the UKPP domain.
-                **orogenh_standard** (iris.cube.Cube):
-                    Precipitation enhancement due to orography in mm/h on
-                    the 1km Transverse Mercator UKPP grid domain.
+        iris.cube.Cube:
+            Precipitation enhancement due to orography in mm/h on the
+            UK standard grid, padded with masked up np.nans where
+            outside the UKPP domain.
```

Comment:
This was the wrong way around in the original docstring, but excitingly also wrong in the detail below.  Although we input the orography on the 1km Tranverse Mercator UKPP grid (and we need to in the IMPROVER suite), this is not mandated by the code.  So this docstring should say
```
iris.cube.Cube:
    Precipitation enhancement due to orography on the high
    resolution input orography grid.
```
or equivalent.

Reply:
Done.